find-child-packages.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+const loadJsonFile = require('load-json-file');
+const glob = util.promisify(require('glob'));
+
+const exists = util.promisify(fs.exists);
+
+const PACKAGE_JSON_FILE = 'package.json';
+const LERNA_CONFIG_FILE = 'lerna.json';
+
+const flat = arrays => [].concat.apply([], arrays);
+
+const distinct = array => [ ...new Set(array) ];
+
+const findPatternMatches = async (root, pattern) => {
+    if (pattern.includes('**')) return [];
```

Comment:
I like the consts above for lerna/package.json, could we make this a const as well, to make it clear what this string is?